scss/_buttons.scss

```diff
@@ -232,23 +232,28 @@
 
 // Boosted mod: social button
 .btn-social {
-  border-color: currentcolor;
+  --#{$prefix}btn-border-color: currentcolor;
+  --#{$prefix}btn-hover-color: #{$white};
+  --#{$prefix}btn-hover-bg: var(--#{$boosted-prefix}network-color, $black);
+  --#{$prefix}btn-hover-border-color: var(--#{$boosted-prefix}network-color, $black);
+  --#{$prefix}btn-active-color: #{$white};
+  --#{$prefix}btn-active-bg: #{$black};
```

Comment:
Why not:
- `btn-hover-bg-color` instead of `btn-hover-bg`?
- `btn-active-bg-color` instead of `btn-active-bg`?

Reply:
Because we reuse the CSS var coming from `.btn`. Since they come from Bootstrap, I'd say that it is smaller so more understandable 🤔